notifications/src/main/java/org/opennms/horizon/notifications/service/MonitoringPolicyService.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.horizon.notifications.service;
+
+import org.opennms.horizon.notifications.mapper.MonitoringPolicyMapper;
+import org.opennms.horizon.notifications.model.MonitoringPolicy;
+import org.opennms.horizon.notifications.repository.MonitoringPolicyRepository;
+import org.opennms.horizon.notifications.tenant.WithTenant;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.opennms.horizon.shared.alert.policy.MonitorPolicyProto;
+
+@Service
+public class MonitoringPolicyService {
+
+    @Autowired
+    private MonitoringPolicyMapper monitoringPolicyMapper;
+
+    @Autowired
+    private MonitoringPolicyRepository monitoringPolicyRepository;
```

Comment:
Nit: I tend to prefer constructor injection

```suggestion
@RequiredArgsConstructor
@Service
public class MonitoringPolicyService {

    private final MonitoringPolicyMapper monitoringPolicyMapper;

    private final MonitoringPolicyRepository monitoringPolicyRepository;
```

Reply:
Agreed, just following existing style in this service, but could have a task to rework those.